src/kristal.lua

```diff
@@ -217,13 +217,16 @@ function love.load(args)
         TARGET_MOD = Kristal.Args["mod"][1]
     end
 
-    if not love.filesystem.getInfo("mario.png") then
+    if not (love.filesystem.getInfo("mario.png", "file") or love.filesystem.getInfo("marios", "directory")) then
         local main_font = love.graphics.newFont("assets/fonts/main.ttf", 32)
         Gamestate.switch({
             draw = function()
                 love.graphics.setColor(1, 1, 1)
                 love.graphics.setFont(main_font)
-                love.graphics.printf("Please insert mario.png into the base Kristal folder and restart", 0, 0, SCREEN_WIDTH)
+                love.graphics.printf([==[
+Please insert mario.png into the base Kristal folder and restart
+Alternatively, create a "marios" directory containing images.
```

Comment:
```suggestion
Alternatively, create a "marios" directory containing mario images.
```
Please specify. They cannot just be any image

Reply:
seconding this